src/configuration/remapper.ts

```diff
@@ -480,6 +483,21 @@ export class Remapper implements IRemapper {
 
     vimState.recordedState.resetCommandList();
     if (remapping.after) {
+      let changeToVisualForRemap: Mode | undefined;
+      let selectedRange: vscode.Range | undefined;
+      let selectedText: string | undefined;
+      if (this._configKey === 'allVisualModeKeyBindingsMap' && isSelectMode(vimState.currentMode)) {
+        // Remaps from 'allVisualModeKeyBindingsMap' (VIM's vmap) when ran in select modes temporarely
+        // change to visual mode so that the execution is similar to the visual mode
+        // (:help select-mode-mapping)
+        changeToVisualForRemap = vimState.currentMode;
+        selectedRange = new vscode.Range(
```

Comment:
nit: I think this can just be `selectedRange = vimState.editor.selection;` since IIRC `Selection` is a subclass of `Range`

Reply:
Didn't notice that. 😄